Sources/PT.PM.CSharpParseTreeUst/Converter/Helper.cs

```diff
@@ -35,6 +41,100 @@ protected TypeToken ConvertType(Ust node)
             return null;
         }
 
+        /// <summary>
+        /// Performs deconstruction of the tuples, in the absence of a name, else create vitual memrefs
```

Comment:
"When a name is absent, deconstruct a given tuple, otherwise create virtual memrefs."

What is a "memref" by the way? :)

Reply:
member reference)